Code/Tools/ProjectManager/Source/EditAGemScreen.cpp

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <GemCatalog/GemCatalogHeaderWidget.h>
+#include <ProjectUtils.h>
+#include <PythonBindingsInterface.h>
+#include <ScreenHeaderWidget.h>
+#include <EditAGemScreen.h>
+
+#include <QApplication>
+#include <QButtonGroup>
+#include <QComboBox>
+#include <QDialogButtonBox>
+#include <QDir>
+#include <QLabel>
+#include <QMessageBox>
+#include <QPushButton>
+#include <QRadioButton>
+#include <QScrollArea>
+#include <QStackedWidget>
+#include <QVBoxLayout>
+
+
+namespace O3DE::ProjectManager
+{
+
+    EditGem::EditGem(QWidget* parent)
+        : CreateGem(parent)
+    {
+        //undo the connections of the parent class
+        m_header->backButton()->disconnect();
+        m_backButton->disconnect();
+        m_nextButton->disconnect();
```

Comment:
Another option - instead of disconnecting the parent events, would be to move the parent's calls to `connect` into a function called `Init` or something that is virtual and `EditGem` could override this method and have its own connect methods there.

Reply:
Instead of calling it `Init` we could just call it `HookConnections` so that the purpose is clear. It seems like we can call `connect` at any point in initialization, so long as it's done before the user can start interacting with the application.